lib/engines/hcktest/platforms/Win10_1607x64.json

```diff
@@ -5,7 +5,7 @@
   "clients": {
     "c1": {
       "name": "CL1",
-      "cpus": "2",
+      "cpus": "4",
```

Comment:
Should we have at least 4 CPU for IGB testing? Or why you change the default?

Reply:
Sorry, I missed your commit message. I have 2 comments regarding this. 
1. For NetKVM we skip 1c_Mini6RSSOids by default because it required 16 CPUs. I agree to set 4 CPUs but if Intel changes the driver defaults this test will fail. As a variant, do not touch defaults and just override the end server configuration.
2. Please update the c2 configuration as well. When the HCK test required only one PC but two or more are present controller can use any of them. I see that sometimes "1 Machine" tests are running on c2.

Reply:
Updated c2 configuration.